### DISCUSSION TITLE
Protective gear rework

### DISCUSSION THREAD

**Discussion by IgRAzm** [FakeFishGames/Barotrauma#8751] [Ideas and suggestions]
In my experience, many people agree there's a lot to refine about the protective wearables in this game. My suggestions aim to increase the meaning of gameplay around them; knowing that others probably have a good idea in their head on how variables function in the playing process, I won't explain every change's full reasoning here. I tried to keep close to vanilla balance wherever it doesn't feel like a big change is needed, but all gear in the end was meant to be tweaked to not feel underwhelming to use.

In this post I'll share ideas on the helmets. My thoughts are that this slot is particularly underwhelming, for a few exceptions, and doesn't provide a lot of optimization or strategy.

Headwear - differentiated further; added more purpose in not wearing helmets; lesser value in using helmets under the diving suits.

   **Diving mask**
   
Gunshots: 20%
Bites: 20%
Blunt force: 20%
Husk infection: 40%
Movement: remove -10% debuff
Firearms spread: +15%
Enables breathing when filled with air tanks

   **Baseball Cap**

No longer a headwear, works as a headset with a different visual for mechanics from the start. Visually stays on when using diving mask, thermal goggles or health scanner.

   **Captain's Hat (all three types)**

Gunshots: 30%
Bites: 20%
Blunt force: 30%
Burns: 20%
Lacerations: 20%

Helm bonus removed, instead added a passive, "Coordination":
Each time a Helm skill point is gained, allied players in sight without any outerwear, including the cap, receive full-body 25% protection against abovementioned damage types for 30 seconds; this resist bonus is doubled if the crew member has no headwear.

   **Riot Helmet**

Gunshots: 20%
Bites: 40%
Blunt force: 40%
Burns: 20%
Lacerations: 30%
Bleeding: 30%
Movement: -10%
Concussion: 45%

New passive, "Complete Kit":
Resistances on this headwear are doubled if used together with indoors combat armor.

   **Ballistic Helmet**

Gunshots: 40%
Bites: 25%
Blunt force: 40%
Burns: 20%
Lacerations: 20%
Bleeding: 20%
Movement: -10%
Concussion: 45%

Passive "Complete Kit".

**Gunner's Helmet**

Gunshots: 50%
Bites: 40%
Blunt force: 50%
Burns: 20%
Lacerations: 40%
Bleeding: 40%
Concussion: 75%

Passive, "Periscope-Compatible": +10 to Weapons unless worn inside a diving suit.

   **New Headwear:**

![helmet-1](https://user-images.githubusercontent.com/97034923/162536034-fe5abb8c-4629-45f0-bdb5-2c70f6c47fe4.jpg)

   **Bonnet Diving Helmet** Crafting (Mechanical 30): Diving Mask, Copper, Carbon, Rubber
"Old-style diving helmet made of copper and reinforced with modern carbon fiber. Not recommended for diving, but can halt the pressure against your skull and upper body for a little while during a hull breach."

Gunshots: 40%
Bites: 40%
Blunt force: 40%
Burns: 40%
Lacerations: 40%
Bleeding: 40%
Concussion: 80%
Husk infection: 60%
Firearms spread: +25%
Movement: -15%
Enables breathing when filled with air tanks
Pressure: protects up to 3000 meters deep, but in return you will start to receive slowly increasing internal damage after a minute or two exposed.

Passive, "Antique Helmet":
The body is protected from pressure up to an extent. When worn under a diving suit, the item gives halved bonuses. 

![png-transparent-firefighter-mask-fire-department-firefighter-photography-people-helmet](https://user-images.githubusercontent.com/97034923/162536016-ada48b93-89f5-4887-b9f7-bb98f077f0b3.png)

   **Fireman Helmet** Crafting (Engineering 30) Diving Mask, Plastic, Steel Bar
"Widely used back on Earth once, protection like this isn't valued any less here, as fires always were and remain a real risk with most of electrical devices on Europa."

Gunshots: 20%
Bites: 20%
Blunt force: 50%
Burns: 70%
Husk infection: 40%
Movement: -10% debuff; -20% when in water
Firearms spread: +20%
Enables breathing when filled with air tanks

Passive, "Fireman":
Increases fire extinguisher range when the item isn't worn under a diving suit.

   Anyway, I hope others see potential in the ideas. I really think something should eventually be done about the clothes and such, maybe not with this execution but still, changes that even out the viability of choices.

**Comment by mLuby** [comment]
I like your idea for passive "complete kit" set bonuses, basically that donning both headgear and bodywear that share a tag (e.g. `firefighter` or `clown`) triggers some additional benefits from one or both items.

It's a nice way of encouraging (but not forcing) people to wear items that make sense together, instead of ugly hodgepodges like a red diving helmet and a green diving suit body.

<hr />

I'm not a fan of each item having small resistances to everything like this since it clogs the UI and muddies player choices:
```
Gunshots: 20%
Bites: 20%
Blunt force: 50%
Burns: 70%
Husk infection: 40%
Movement: -10% debuff; -20% when in water
Firearms spread: +20%
```
Instead I'd like to see items focus on doing one thing well (and maybe one significant negative), like this:
```
Burns: 70%
```

<table><tbody><tr><td width="50%">
<p>Diving (hard)suits present two unique problems:</p><ol><li>When _not_ in the extreme environment for which they're designed, suits would necessarily be very effective at stopping most forms of damage; really everything except except armor-piercing projectiles. Fire or acid could destroy seals but it'd take a few seconds at least.</li><br/><li>When actually _in_ the ocean's intense pressure, it'd only take a tiny bit of damage to crack the suit and expose the diver to acute barotrauma and death.</li></ol>
</td><td width="50%"><img alt="Fearful diver inside cracked helmet" src="https://user-images.githubusercontent.com/2483420/162553355-5f80bf80-70ae-4139-b2ae-7a6e8c45340a.png" /></td></tr></tbody></table>

My suggestion here is to _make the suit take damage_ instead of the character, and once the suit's condition is low enough it can break, removing its pressure immunity affliction and passing further damage along to the wearer. Body armor, helmets, and riot shields could take damage in the same way but the consequences would be most dramatic for suits.

Also, if they're calibrated for fluid movement in dense water, it'd stand to reason that _diving suits should be clumsy to control when out of water._ That would mean you'd have a hard time controlling movement or aiming effectively when in air, thus encouraging players to wear appropriate gear for their surroundings.

As an aside, if separating out diving helmet from diving suit (body), then `pressure immunity` needs to be a set bonus lest you get someone wearing half a diving suit without the other. This might confuse newer players and will definitely confuse bots. It also looks quite ugly if mixing parts from different diving suits like red helmet and green suit, though that could be mitigated by keeping set bonuses just between matching suit parts.

**Reply by IgRAzm** [reply]
Appreciate the detailed response.
The small overall bonuses on armor are a stay over from the existing balance pretty much. I changed them but wanted to have some ups and down on each. You got to know how popular helmets are in serious matches; a fireman helmet with just fire res doesn't compete well with them (and imo it makes sense mechanically that it atl as strong as Cap's Hat). Also wanted my diving masks have a reason to be used too.

I wasn't really clear at times so you mb extrapolated some changes beyond where I intended. Diving suits remain in their full piece form, it's just that I want a better diving mask for specific cases when it can be a better choice than a suit; you get to keep your body armor is what I aimed for. Also if we go from realism direction, I think this is relatively within bounds of possible - just have to headcanon that the inner clothes are worn above a built in helmet rubber suit that covers torso and limbs.

About the realistic diving suit functions - I definitely won't hasten to say I don't like it, possibly it would improve both immersion and pressure gameplay quality. But, my thread focused on things easy to develop; in extreme case my ideas can be used for a mod, and I don't think game is currently moddable to make diving suits like this. It's not like our approaches are too different, I added weapon inaccuracy to the masks and such and also want indoors and outdoors equipment used appropriately.
